l10n_it_riba_oca/tests/test_riba.py

```diff
@@ -278,7 +278,7 @@ def test_riba_incasso_all_paid(self):
         self.invoice.action_post()
         self.assertEqual(self.invoice.state, "posted")
 
-        to_issue_action = self.env.ref("l10n_it_riba.action_riba_to_issue")
+        to_issue_action = self.env.ref("l10n_it_riba_oca.action_riba_to_issue")
         to_issue_model = self.env[to_issue_action.res_model]
         to_issue_domain = safe_eval.safe_eval(to_issue_action.domain)
         to_issue_records = (
```

Comment:
Qui sotto (riga 334) c'è ancora un riferimento a `l10n_it_riba`, puoi rinominarlo in `l10n_it_riba_oca`?
È il motivo per cui stanno fallendo i test.